test/configure.js

```diff
@@ -0,0 +1,301 @@
+var path = require( 'path' );
+
+var sugarcoat = require( '../index' );
+
+suite( 'Configure: Settings', () => {
+
+    test( 'Destination is set to be required. Destination errored out when not supplied.', (done) => {
+
+        var configMissingDest = {
+            sections: [
+                {
+                    title: 'CSS File',
+                    files: './test/assert/parseVarCode.css'
+                },
+                {
+                    title: 'CSS File 2',
+                    files: './test/assert/parseVarCode.css'
+                }
+            ]
+        };
+
+        sugarcoat( configMissingDest )
+        .then( data => {
+
+            done();
+        }, data => {
+
+            assert.isArray( data, 'Sugarcoat returns an array.' );
+            assert.propertyVal( data[0], 'key', 'settings.dest', 'We have dest in the error array');
+            done();
+        });
+
+    });
+
+    test( 'Destination can be set to none. No index file is created.', done => {
+
+        var configNoDest = {
+            settings: {
+                dest: 'none'
+            },
+            sections: [
+                {
+                    title: 'CSS File',
+                    files: './test/assert/parseVarCode.css'
+                },
+                {
+                    title: 'CSS File 2',
+                    files: './test/assert/parseVarCode.css'
+
+                }
+            ]
+        };
+
+        sugarcoat( configNoDest )
+        .then( data => {
+
+            assert.isNull( data.settings.dest, 'The destination was set to null' );
+            assert.isFalse( fs.existsSync( path.relative( data.settings.cwd, 'index.html' ) ), 'Sugarcoat did not create a index.html file.' );
+            done();
+        });
+    });
+
+    teardown( done => {
+
+        fs.remove( './sugarcoat', err => {
+
+            if ( err ) return console.error( err );
+
+            done();
+        });
+
+    });
+});
+
+suite( 'Configure: Sections', () => {
+
+    test( 'Title is set to be required. Title errored out when title is the only required option that was not supplied.', done => {
+
+        var configMissingOnlyTitle = {
+            settings: {
+                dest: './test/documentation',
+                title: 'Pattern Library'
+            },
+            sections: [
+                {
+                    files: './test/assert/parseVarCode.css'
+                }
+            ]
+        };
+
+        sugarcoat( configMissingOnlyTitle )
+        .then( data => {
+
+            done();
+        }, data => {
+
+            assert.isArray( data, 'Sugarcoat returns an array.' );
+
+            if ( data.length === 1 ) {
+
+                assert.propertyVal( data[0], 'key', 'sections.title', 'We have sections.title in the error array.' );
+            }
+
+            done();
+        });
+    });
+
+    test( 'Title is set to be required. Title errored out when title and dest were not supplied.', done => {
```

Comment:
We've already tested when dest is not supplied. For unit tests, we usually want to cover one change at a time, and not an accumulation of small changes.